src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalController.java

```diff
@@ -137,7 +137,7 @@
     @GetMapping(produces = APPLICATION_JSON_VALUE)
     public ResponseEntity<Object> retrieveOrganisations(
             @Pattern(regexp = ORGANISATION_IDENTIFIER_FORMAT_REGEX, message = ORG_ID_VALIDATION_ERROR_MESSAGE)
-            @ApiParam(name = "id", required = false) @RequestParam(value = "id", required = false) String id,
+            @ApiParam(name = "id") @RequestParam(value = "id", required = false) String id,

```

Comment:
is this part of S2S changes ? I think this PR should be only for S2S changes , for any minor controller level changes, complete regression testing will be needed.

Reply:
This is to address sonar smells, `required = false` is the default for `@ApiParam`

Reply:
Got to know these are Sonar fixes, please have regression tested in PR before merging in master

Reply:
have requested @mallikarjunputtana to regression test before merge